src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -337,6 +337,11 @@ private static String getDBConnectionName(String dbname, JSONObject options) {
 
             this.q = new LinkedBlockingQueue<DBQuery>();
             this.openCbc = cbc;
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+                Log.v(SQLitePlugin.class.getSimpleName(), "Applying hotfix for Android 11+");
+                this.oldImpl = true;
```

Comment:
Can we have an explanation on how this affects the resolution? maybe a github issue link or something on the commit message?

Reply:
We have this one, but it doesn't refer the `oldImpl` field.

https://github.com/storesafe/cordova-sqlite-storage-dependencies/issues/3

Reply:
The `this.oldImpl = true`  is the same result of using the mentioned workaround `androidDatabaseProvider: 'system'`.

Reply:
I will add more context to the commit message.

Reply:
Done